tests/bench.py

```diff
@@ -8,21 +8,43 @@
 
 
 def benchmark_1():
-    x = np.random.randn(10000, 100) * 0.01
+    x = np.random.randn(10000, 1000) * 0.01
     idx = pd.date_range('1990-01-01', freq='B', periods=x.shape[0])
     data = np.exp(pd.DataFrame(x, index=idx).cumsum())
 
     s = bt.Strategy('s', [bt.algos.RunMonthly(),
-                          bt.algos.SelectAll(),
```

Comment:
double check

Reply:
The two algos back to back are actually duplicative. IF nothing is selected, then SelectRandomly will start by doing essentially what SelectAll does. The main reason I removed it is because both algos were actually performing data cleaning (i.e. removing records with no data or negative data), and this was reasonably slow, so doing it twice was taking up a significant amount of time, which hides the performance changes in the underlying engine. This way it's only done once, and it's easier to see how changes to core.py impact performance.